### PR TITLE
[EDU-3995] feat: add zone and account ctas

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added create account and create zone CTAs with buttons.

[EDU-3995]: https://aziontech.atlassian.net/browse/EDU-3995?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/guides/intelligent-dns/run-dig-command.mdx [hunk 1]
```diff
@@ -81,7 +89,13 @@ The `+trace` option is an instruction to return the full path that the DNS serve
 
 This response is quite different from the simple request and presents all DNS servers involved in the resolution of a domain, passing through the Top-Level Domain (TLD) servers until reaching the last authoritative server.
 
-Checking the latency of a web application with the dig command is one of the ways of validating Azion's Edge Computing service. For example, you can compare the latency of your application, served by cloud and by Azion Edge Computing platform.
+### Using dig to test Azion zones and edge applications
+
+If you've [configured a zone](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/), you can use the dig command to test how and if it's receiving proper traffic. For example, if you've configured a load balance with DNS record, you can test and see if the load balance is working properly.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/intelligent-dns/run-dig-command.mdx:94]:
```suggestion
If you've [configured a zone](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/), you can [use the dig command](/en/documentation/products/guides/run-the-dig-command/) to test how and if it's receiving proper traffic. For example, if you've configured a load balance with DNS record, you can test and see if the load balance is working properly.
```

File: src/content/docs/pt-br/pages/guias/intelligent-dns/run-dig-command.mdx [hunk 2]
```diff
@@ -81,7 +87,13 @@ A opção `+trace` é uma instrução para retornar o caminho completo que o ser
 
 Essa resposta é bem diferente da requisição simples e apresenta todos os servidores DNS envolvidos na resolução de um domínio, passando pelos servidores do *Top-Level Domain (TLD)* até chegar no último servidor autoritário
 
-Checar a latência de uma aplicação web com o comando dig é uma das formas de validação do serviço de Edge Computing da Azion. É possível, por exemplo, comparar a latência da sua aplicação, servida por um serviço de Cloud Computing e pela Edge da Azion.
+### Use o dig para testar zonas e edge applications na Azion
+
+Se você [configurou uma zona,](/pt-br/documentacao/produtos/guias/secure/intelligent-dns-definir-main-settings/), você pode usar o comando dig para testar como e se ela está recebendo o tráfego adequado. Por exemplo, se você configurou um balanceamento de carga com registro DNS, você pode testar e ver se o balanceamento de carga está funcionando corretamente.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/intelligent-dns/run-dig-command.mdx:92]:
```suggestion
Se você [configurou uma zona](/pt-br/documentacao/produtos/guias/secure/intelligent-dns-definir-main-settings/), você pode usar o comando dig para testar como e se ela está recebendo o tráfego adequado. Por exemplo, se você configurou um balanceamento de carga com registro DNS, você pode testar e ver se o balanceamento de carga está funcionando corretamente.
```

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/guides/intelligent-dns/run-dig-command.mdx [hunk 3]
```diff
@@ -81,7 +89,13 @@ The `+trace` option is an instruction to return the full path that the DNS serve
...
+
+<Button href="/en/documentation/products/guides/secure/load-balance-dns/#testing-your-balanced-record" text="test balanced record" variant="secondary"></Button>
+
+You can also check the latency of a web application with the dig command is one of the ways of validating Azion Edge Computing service. For example, you can compare the latency of your application, served by cloud and by Azion Edge Computing platform.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/intelligent-dns/run-dig-command.mdx:98]:
```suggestion
You can also check the latency of a web application with the dig command to validate the Azion Edge Computing service. For example, you can compare the latency of your application, served by cloud and by Azion Edge Computing platform.
```

File: src/content/docs/pt-br/pages/guias/intelligent-dns/run-dig-command.mdx [hunk 4]
```diff
@@ -81,7 +87,13 @@ A opção `+trace` é uma instrução para retornar o caminho completo que o ser
...
+
+<Button href="/pt-br/documentacao/produtos/guias/secure/balanceamento-de-carga-dns/" text="teste o registro balanceado" variant="secondary"></Button>
+
+Você também pode checar a latência de uma aplicação web com o comando dig é uma das formas de validação do serviço de Edge Computing da Azion. É possível, por exemplo, comparar a latência da sua aplicação, servida por um serviço de Cloud Computing e pela Edge da Azion.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/intelligent-dns/run-dig-command.mdx:96]:
```suggestion
Você também pode checar a latência de uma aplicação web com o comando dig para validar o serviço de Edge Computing da Azion. É possível, por exemplo, comparar a latência da sua aplicação, servida por um serviço de Cloud Computing e pela Edge da Azion.
```

**Review by bru-andrade — APPROVED** [review]

File: src/content/docs/en/pages/guides/intelligent-dns/run-dig-command.mdx [hunk 5]
```diff
@@ -6,10 +6,18 @@ namespace: documentation_how_to_troubleshoot_dig
 permalink: /documentation/products/guides/run-the-dig-command/
 ---
 
+import Button from '~/components/Button.astro'
+
 **Dig** or **dig command** is a command-line tool for querying domain information on Domain Name System (DNS) servers. Originally called *Domain Information Groper*, the command is part of BIND, a group of networking tools maintained by the Internet Systems Consortium (ISC).
 
 The `dig` command is commonly used for network and domain troubleshooting, and also other DNS operations. It's also a multiplatform and an alternative to the discontinued `nslookup`.
 
+If you have a zone configured in **Azion Intelligent DNS**, you can test it to see if your zone is working properly.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/intelligent-dns/run-dig-command.mdx:15]:
```suggestion
If you have a zone configured in **Azion Intelligent DNS**, you can test it to check if your zone is working properly.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/guides/intelligent-dns/run-dig-command.mdx:15]: or confirm/validate

File: src/content/docs/en/pages/guides/intelligent-dns/run-dig-command.mdx (hunk 1 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/intelligent-dns/run-dig-command.mdx:94]:
```suggestion
If you've [configured a zone](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/), you can use the dig command to test how and if it's receiving proper traffic. For example, if you've configured a load balance with a DNS record, you can test and check if the load balance is working properly.
```

File: src/content/docs/pt-br/pages/guias/intelligent-dns/run-dig-command.mdx [hunk 6]
```diff
@@ -1,16 +1,22 @@
...
-
 ---
 
+import Button from '~/components/Button.astro'
+
 **Dig** ou **comando dig** é uma ferramenta de linha de comando para consulta de informações sobre domínios em servidores Domain Name System (DNS). Originalmente chamado de Domain Information Groper, o comando faz parte do BIND, um grupo de ferramentas de rede mantido pelo Internet Systems Consortium (ISC).
 
 O comando `dig` é comumente utilizado para solução de problemas de rede, domínios e outras operações envolvendo DNS. É, também, uma alternativa multiplataforma e mais poderosa ao descontinuado `nslookup`.
 
+Se você tiver uma zona configurada no **Azion Intelligent DNS**, você pode testá-la para ver se sua zona está funcionando corretamente.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/intelligent-dns/run-dig-command.mdx:14]:
```suggestion
Se você tiver uma zona configurada no **Azion Intelligent DNS**, você pode testá-la para conferir se sua zona está funcionando corretamente.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**